cmd/crowdsec-cli/console.go

```diff
@@ -187,9 +187,17 @@ Disable given information push to the central API.`,
 	cmdConsole.AddCommand(cmdDisable)
 
 	cmdConsoleStatus := &cobra.Command{
-		Use:               "status [feature-flag]",
-		Short:             "Shows status of one or all feature flags",
-		Example:           "status tainted",
+		Use:   "status [feature-flag]",
+		Short: "Shows status of one or all feature flags",
+		Example: `sudo cscli console status
++-------------+-----------+---------------------------------------------------+
```

Comment:
We do not have any output example in `cscli`, and the output can be somehow confusing when running `cscli console status -h`

Reply:
Another thing - we use the term "feature-flags" for both console and crowdsec/cscli. This is confusing especially when I merge "cscli config feature-flags". What can we do? @buixor ?

Reply:
Removed the output not to confuse people. Agreed with @mmetc on the feature-flag confusion; hence replaced with "option" and "console option" in help messages. I believe it's more accurate and less confusing